pr_agent/algo/utils.py

```diff
@@ -304,3 +304,18 @@ def try_fix_yaml(review_text: str) -> dict:
         except:
             pass
     return data
+
+
+async def set_custom_labels(variables):
```

Comment:
why async ?

Reply:
Because it is called from an async function